lms/static/js/bookmarks/views/bookmarks_list.js

```diff
@@ -0,0 +1,133 @@
+;(function (define, undefined) {
+    'use strict';
+    define(['gettext', 'jquery', 'underscore', 'backbone', 'moment'],
+        function (gettext, $, _, Backbone, _moment) {
+
+        var moment = _moment || window.moment;
+
+        return Backbone.View.extend({
+
+            el: '.courseware-results-list',
+            coursewareContentElement: '#course-content',
```

Comment:
Actually, on second thoughts `coursewareContentEl` would be more consistent with backbone conventions.